apps/antalmanac/src/components/RightPane/SectionTable/EnrollmentHistoryPopup.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState, useEffect, useMemo } from 'react';
+import { LineChart, Line, CartesianGrid, ResponsiveContainer, XAxis, YAxis, Tooltip, Legend } from 'recharts';
+import { Box, Link, Typography, Skeleton, useMediaQuery } from '@mui/material';
+import { MOBILE_BREAKPOINT } from '../../../globals';
+import { DepartmentEnrollmentHistory, EnrollmentHistory } from '$lib/enrollmentHistory';
+import { isDarkMode } from '$lib/helpers';
+
+export interface EnrollmentHistoryPopupProps {
+    department: string;
+    courseNumber: string;
+}
+
+const EnrollmentHistoryPopup = ({ department, courseNumber }: EnrollmentHistoryPopupProps) => {
+    const [loading, setLoading] = useState(true);
+    const [enrollmentHistory, setEnrollmentHistory] = useState<EnrollmentHistory>();
+    const isMobileScreen = useMediaQuery(`(max-width: ${MOBILE_BREAKPOINT})`);
+
+    const deptEnrollmentHistory = useMemo(() => new DepartmentEnrollmentHistory(department), [department]);
+
+    const graphWidth = useMemo(() => (isMobileScreen ? 250 : 450), [isMobileScreen]);
+    const graphHeight = useMemo(() => (isMobileScreen ? 175 : 250), [isMobileScreen]);
+    const popupTitle = useMemo(() => {
+        if (!enrollmentHistory) {
```

Comment:
Purely stylistic preference, and I can see both ways.
Clarifying that the condition is a null check instead of a boolean check can be helpful, but that adds extra characters to read.

```suggestion
        if (enrollmentHistory == null) {
```